crypto/bio/b_print.c

```diff
@@ -367,13 +364,18 @@ fmtstr(char **sbuffer,
        size_t *maxlen, const char *value, int flags, int min, int max)
 {
     int padlen;
-    size_t strln;
+    int strln;
     int cnt = 0;
 
     if (value == 0)
         value = "<NULL>";
 
-    strln = OPENSSL_strnlen(value, max < 0 ? SIZE_MAX : (size_t)max);
+    /*
+     * TODO: INT_MAX was changed to SIZE_MAX in by Richard Levitte
+     *       in commit 230c691a521 (2016-05-21 376).
+	 *       This looks like an accidental change to me.
+     */
+    strln = (int)OPENSSL_strnlen(value, max < 0 ? SIZE_MAX : (size_t)max);
 
```

Comment:
TODO: INT_MAX was changed to SIZE_MAX in by @levitte in commit 230c691a521 (2016-05-21 376). This looks like an accidental change to me.